p4_symbolic/bmv2/bmv2.proto

```diff
@@ -25,6 +25,37 @@ package p4_symbolic.bmv2;
 
 import "google/protobuf/struct.proto";
 
+// Acceptable values for match type of keys of a table.
+enum TableMatchTypeEnum {
+  // Casing must match the bmv2 json!
+  exact = 0;
+  lpm = 1;
+  ternary = 2;
+  range = 3;
+}
+
+// The type (of the match) of a parser transition.
+enum ParserTransitionTypeEnum {
+  // The default transition (no matching).
+  default = 0;
+  // Transition is applied when hex strings match.
+  // The match hex string and the matched field are defined
+  // in ParserTransition.
+  hexstr = 1;
+  // Matching via a named parse value set.
+  // The name of the value set is defined in ParserTransition.
+  parse_vset = 2;
+}
+
+// The type of a table.
+// Determines where actions are defined and how they get selected.
+// This is always "simple" in all our sample programs.
```

Comment:
Remove this comment. It won't stay true potentially, and it has nothing to do with the enum.